PopcornPals-Server/src/main/java/bg/fmi/popcornpals/dto/StudioDTO.java

```diff
@@ -0,0 +1,35 @@
+package bg.fmi.popcornpals.dto;
+
+import bg.fmi.popcornpals.util.RegexPattern;
+import bg.fmi.popcornpals.util.StringSize;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class StudioDTO {
+
+    @NotNull(message = "Studio: Name cannot be null")
```

Comment:
No need for validation in the DTOs

Reply:
As far as i am aware its more important for the DTO to have validatian rather thatn the model